tutorials/tracing/manual_instrumentation_tutorial.ipynb

```diff
@@ -0,0 +1,784 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    span_exporter = OTLPSpanExporter(endpoint=f"http://{COLLECTOR_HOST}:6006/v1/traces")
</pre><p>I would just explicitly set the collector host to localhost.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/2836/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>